src/core/utils/helpers/general.ts

```diff
@@ -47,6 +54,11 @@ export const flattenCreators = (creators: Work["creators"]) =>
     return creator.display;
   });
 
+export const flattenCreatorsLastNameFirst = (creators: Work["creators"]) =>
+  creators.map((creator: Work["creators"][0]) => {
```

Comment:
I dont think it is necessary to define the creators type again. TS is capable of inferring that by itself:
```suggestion
  creators.map((creator) => {
```